### PR TITLE
Contact bot and bot wizard will be visible only if user is logged in

### LINKED ISSUE #704: Contact Bot and Bot wizard pages shouldn't be visible when user isn't logged in
**Actual Behaviour**

Contact Bot page (https://skills.susi.ai/botbuilder/contactbot) and Bot Wizard page (https://skills.susi.ai/botbuilder/botwizard) are visible even if the user isn't logged in.

**Expected Behaviour**

They shouldn't be.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by DNS-404**
Fixes #704 

Changes: Contact bot and bot wizard aren't visible now if user isn't logged in.

Surge Deployment Link: https://pr-706-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

**Comment by DNS-404** [comment]
@hkedia321 @anshumanv @Akshat-Jain @akshatnitd @PrP-11 Please review.

File: src/components/BotBuilder/ContactBot.js [hunk 1]
```diff
@@ -18,6 +18,19 @@ class ContactBot extends React.Component {
...
 
 	render() {
+		if(!cookies.get('loggedIn'))
+        {
+            return (
+                <div>
+                    <StaticAppBar {...this.props} />
+                    <div>
+                        <p style={styles.loggedInError}>
+                            Please login create the Contact Bot.
```
Comment (Akshat-Jain) [inline src/components/BotBuilder/ContactBot.js]: Make it "Please login to create the Contact Bot" :sweat_smile:

Comment (DNS-404) [reply src/components/BotBuilder/ContactBot.js]: Thanks. Made the change.

**Comment by AnupKumarPanwar** [comment]
Please resolve conflicts

**Comment by DNS-404** [comment]
Resolved the conflicts

**Pull request merged**